Makefile

```diff
@@ -1,22 +1,23 @@
-
+DESTDIR ?= 
+PREFIX ?= /usr/local/ 
 BUILDDIRS=src lib tool doc test
 
 .PHONEY: all install clean distclean
 
 all:
-	@$(foreach dir,$(BUILDDIRS),make -C '$(dir)' all && ) echo Built all
+	@$(foreach dir,$(BUILDDIRS),make DESTDIR=$(DESTDIR) PREFIX=$(PREFIX) -C '$(dir)' all && ) echo Built all
```

Comment:
I never saw the form "make VAR=$(VAR)" -- can you point me to the GNU Makefile documentation describing this? (I looked, but failed to find it.)